SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -417,6 +425,35 @@ void USpatialWorkerConnection::ProcessOutgoingMessages()
 		}
 		}
 	}
+
+	// Flush worker API calls
+	if (bSentData)
+	{
+		Worker_Connection_Alpha_Flush(WorkerConnection);
+	}
+}
+
+void USpatialWorkerConnection::MaybeFlush()
+{
+	const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
+	if (Settings->bWorkerFlushAfterOutgoingNetworkOp)
```

Comment:
If this boolean is not set when does `Flush()` get called? Never? If never, what else causes `ProcessOutgoingMessages` to be called?

Reply:
If this isn't set then we rely on a timer to wake the background thread and flush, we call MaybeFlush after RPC calls and replication to flush them out at a faster interval.